Automattic/sensei
# Create and register the patterns for course list block

Implements https://github.com/Automattic/sensei/issues/5423

### Changes proposed in this Pull Request

* Added 6 patterns for course list block

### Testing instructions

- Add a Course List block in the block editor
- Click on 'Choose' to open the pattern selection modal
- Make sure along with the existing patterns, 6 new patterns are there as per our design. 3 of them should be of Grid structure, 3 of them should be of list structure.

**Course Signup and Course Progress block is not currently available in the Page editor. Making the Course Progress block available in the Page editor is in progress and we'll replace the Course Signup block with a new one as part of our upcoming tasks. So for now it's better to test these patterns in the course editor GB page**

![image](https://user-images.githubusercontent.com/6820724/183763851-7ec0387f-a245-4af3-9464-76f305773ac7.png)


### Screenshot / Video

https://user-images.githubusercontent.com/6820724/183763732-5cf6826d-322d-456a-b1a4-093f1c083c03.mov
